compiler/damlc/tests/BUILD.bazel

```diff
@@ -736,15 +758,18 @@ da_haskell_test(
 da_haskell_test(
     name = "repl-functests",
     srcs = ["src/DA/Test/Repl/FuncTests.hs"],
-    data = [
-        ":repl-test.dar",
-        ":repl-test-two.dar",
-        ghc_pkg,
-        "//compiler/damlc/pkg-db",
-        "//compiler/damlc/stable-packages",
-        "//compiler/repl-service/server:repl_service_jar",
-        "//daml-script/daml:daml-script.dar",
-    ] + (["@sysctl_nix//:bin/sysctl"] if is_darwin else []),
+    data =
+        [f for major in LF_MAJOR_VERSIONS for f in [
```

Comment:
python!
later we should make these into per-version tests but it's fine for now

Reply:
Yeah, I couldn't find a better way of doing this with the primitives exposed by bazel.

Reply:
yes sorry to be clear I was complaining about bazel/starlark/python